docs/tutorials/elasticdl_on_prem_cluster.md

```diff
@@ -12,7 +12,7 @@ export TRAVIS_BUILD_DIR=$PWD
 bash scripts/travis/build_images.sh
 ```
 
-## Submit job to cluster
+## Submit Job to Cluster
```

Comment:
The train command should be updated too.
We use --image_name directly.
--image_base and --docker_image_prefix are not necessary anymore.

Reply:
Changed the submit script.